src/collections/_documentation/error-reporting/getting-started-verify/csharp.md

```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
+
```

Comment:
```suggestion
```

Gotta remove both lines (incl. empty line)... let's try out the brand new multi-line suggestions in GitHub :)